### ISSUE TITLE
Discrepancy Between Hyperparameters in Paper and Code — Which Should Be Used for Reproducing Results?

### ISSUE THREAD

**Issue by jiangjiadi** [nusnlp/FSPO#8]
Hi, I noticed some inconsistencies between the hyperparameter values (e.g. learning rate) reported in the paper and those used in the provided code. To faithfully reproduce the results described in the paper, should I follow the values in the paper or the ones in the code? It would be greatly appreciated if you could clarify which configuration was actually used in the experiments reported in the paper. Thanks!
